Makefile

```diff
@@ -12,12 +12,13 @@ MKDIR_P ?= mkdir -p
 # Jenkins overwrites CPPFLAGS, so use CPPARGS instead.
 CPPARGS ?= $(INC_FLAGS) $(CPP_FLAGS) -MMD -MP -DVERSION="\"$(VERSION)\""
 
-m2cell: run_tests $(BIN_DIR)/m2cell
-
+# TODO: DM-  Make the executable in the 'serverMain' directory and copy it to bin.
```

Comment:
What is the ticket number of this `DM-`?

Reply:
DM-35023